docroot/profiles/custom/cgov_site/modules/custom/cgov_image/cgov_image.module

```diff
@@ -27,7 +28,6 @@ function cgov_image_media_view(array &$build, EntityInterface $entity, EntityVie
   if ($entity->bundle() != 'cgov_image' && $entity->bundle() != 'cgov_contextual_image') {
```

Comment:
This method should check if $entity is instanceof MediaInterface. Then all the internal calls can just require MediaInterface in the signature.